ui/src/views/Quickstart/index.jsx

```diff
@@ -29,16 +29,14 @@ import githubQuery from './github.graphql';
 
 const initialYaml = {
   version: 1,
-  tasks: [
-    {
-      provisionerId: '{{ taskcluster.docker.provisionerId }}',
-      workerType: '{{ taskcluster.docker.workerType }}',
-      extra: {
-        github: {
-          env: true,
-          events: [],
-        },
-      },
+  policy: {
+    pullRequests: '',
```

Comment:
The initial editor seem to have pullRequests set to `collaborators`, not `''`.

Reply:
Done